spec/acceptance/food_truck_api_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rack/test'
+require_relative '../../app/api'
+
+module FoodTruck
+  RSpec.describe 'Food Truck API' do
+    include Rack::Test::Methods
+
+    def app
+      API.new
+    end
+
+    describe 'provides message' do
+      it 'of listed food trucks' do
+        get '/api'
+        expect(last_response.status).to eq(200)
```

Comment:
It would be nice to have some stronger expectations here.  For example, if I replace your sinatra route logic with this:

``` ruby
get "/api" do
end
```

...this spec still passes.  I recommend you take a two-part approach to testing this:

- Write a unit spec (similar to the unit spec chapter in the book) that injects a test double in place of the `FoodTruck::SchedulerFetcher` (which I suggested you create above) so that you can have a fast, deterministic unit test with specific, strong expectations about the returned response body.
- Write an acceptance spec that use the real `FoodTruck::SchedulerFetcher` (as a way to ensure everything works together end-to-end when fetching against the real website).  You'll still want some kind of expectations about the response: maybe you can assert something about the general slack format of the output or assert that it returned at least one food truck or something.